src/explicit_ode.pyx

```diff
@@ -202,12 +202,11 @@ cdef class Explicit_ODE(ODE):
         opts["output_list"] = output_list
         opts["output_index"] = 0
         opts["report_continuously"] = 1 if REPORT_CONTINUOUSLY else 0
-        output_index = 0
         
         self.display_progress_activated = 1 if self.display_progress else 0
         self.time_integration_start = timer()
         
-        while (flag == ID_COMPLETE and tevent == tfinal) is False and (self.t-eps > tfinal) if backward else (self.t+eps < tfinal):
+        while (flag == ID_COMPLETE and tevent == tfinal) is False and (self.t - eps*max(abs(self.t), abs(tfinal)) > tfinal) if backward else (self.t + eps*max(abs(self.t), abs(tfinal)) < tfinal):
```

Comment:
By not scaling these `eps` with `time`, they would effectively vanish to sufficiently large `t`, at about `1000`ish

Reply:
Is it possible to put `max(abs(self.t), abs(tfinal))` outside (before) the while-loop or does it need to be part of this line? Feel free to simply resolve this if the answer is yes that it needs to be here (I didn't read all the code in the while-loop).

Reply:
No, since `self.t` is updated within the `while` loop.

Reply:
OK, looks good (resolve)